code/run_iLasso_row.m

```diff
@@ -4,24 +4,26 @@
 p1=params.p1;
 std_dev=params.kernel_width;
 [LX,WX] = size(m,'X');
-numrows = LX;
 if ismember('regix',who(m))
-    numregs = length(m,'regix');
+    regix = m.regix;
+    if iscolumn(regix)
+        regix = regix';
+    elseif ~isrow(regix)
+        display('Error: Regulator indices must be a vector');
+    end
+    numregs = length(regix);
 else
```

Comment:
If `regix` is not defined, revert to the previous behavior where all genes are allowed as regulators